graphql/pupil/fields.ts

```diff
@@ -140,4 +141,12 @@ export class ExtendFieldsPupilResolver {
     async canJoinSubcourses(@Root() pupil: Required<Pupil>) {
         return canJoinSubcourses(pupil);
     }
+
+    @FieldResolver((type) => [PupilScreening])
+    @Authorized(Role.ADMIN, Role.OWNER)
```

Comment:
Is the comment and other "internal stuff" restricted to admins only?

Reply:
I'm not sure, is the newest commit a valid way of restricting the visibility of the field? Or should I create field resolver for the screenings and restrict it there?